packages/devtools-local-toolbox/webpack.config.js

```diff
@@ -29,7 +29,10 @@ module.exports = (webpackConfig, envConfig) => {
     { test: /\.json$/,
       loader: "json" },
     { test: /\.js$/,
-      exclude: /(node_modules|bower_components|fs)/,
+      exclude: request => {
+        return request.match(/(node_modules|bower_components|fs)/)
+               && !request.match(/devtools-local-toolbox/);
+      },
```

Comment:
@juliandescottes I think this will fix a **lot** of the issues you might have been seeing where the toolbox is now typed w/ flow and would break in the console and inspector!